### ISSUE TITLE
Have you heard of streamlit?

### ISSUE THREAD

**Issue by sgbaird** [kjappelbaum/webmofchecker#237]
Recommended to me by @faris-k https://streamlit.io/. Thinking about what my go-to for web app deployment should be (flask, plotly dash, streamlit, etc.)

**Comment by kjappelbaum** [comment]
yes, indeed, many options! If you have a notebook, going via Voila can be the easiest option in many cases (https://github.com/aiidalab/voila-aiidalab-template)

**Comment by sgbaird** [comment]
I looked into it more, and Voila seems pretty nice for what I typically do with Colab notebooks, given that I've been learning `ipywidgets` recently. Took a bit to load one of the examples (due to Binder, makes sense), but after that, pretty slick. Thanks for the share!

https://voila.readthedocs.io/en/stable/using.html
https://github.com/voila-dashboards/voila#rendering-a-notebook-including-interactive-widgets-and-rich-mime-type-rendering
https://voila-gallery.org/
